docs/source/_topic_map.yml

```diff
@@ -10,12 +10,12 @@ Topics:
     Topics:
       - Name: Overview
         File: index
-      - Name: Installing Docker Machine drivers
-        File: docker-machine-drivers
       - Name: Installing Minishift
         File: installing
       - Name: Quickstart
         File: quickstart
+      - Name: Installing Docker Machine drivers
+        File: docker-machine-drivers
```

Comment:
@budhrg User should first go through docker machine driver documentation then should go through minishift installation docs. The original sequence is right IMO. Not sure why you changed it.